pkg/nsm/apis/nseconnect/nseconnect.proto

```diff
@@ -5,66 +5,68 @@ package nseconnect;
 import "github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh/netmesh.proto";
 import "github.com/ligato/networkservicemesh/pkg/nsm/apis/common/common.proto";
 
-// EndpointConnectionRequest is sent by a NSM to NSE to build a connection.
-    message EndpointConnectionRequest {
-        string request_id = 1;
-        string network_service_name = 2;
-    }
-    
+// ConnectRequest is sent
+message ConnectRequest {
+    string request_id = 1;
+    string network_service_name = 2;
+    map<string, string> context = 3;
+    repeated common.LocalMechanism mechanisms = 4;
```

Comment:
Selection of LocalMechanism is done by NSM based on what client requested and what NSE advertised it supported for the specific network service. Then dataplane programs this selection and let know NSE which interface or whatever to use to serve NSE's network service. Not sure why exchanging LocalMechanism back and forth is needed.